easybuild/easyconfigs/c/cdsapi/cdsapi-0.1.4-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'cdsapi'
+version = '0.1.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/wheel'
+description = """A built-package format for Python."""
```

Comment:
yes, wheel is indeed a built-package format for Python... ;)

Reply:
@boegel ping on fixing homepage and descr for this one...